### ISSUE TITLE
End-User Alerts for New Events

### ISSUE THREAD

**Issue by axu-trex** [indico/indico#6050]
I would find it useful if end-users could _subscribe_ to an event category or series, and thus get automatic email reminders when registration opens in any event in said category. Some of our busier attendees would probably even want alerts when event dates are set, so they can block it in their calendar in advance...

**Comment by axu-trex** [comment]
On a related note: is it possible to get indico to send an email to our programme committee mailing list when new abstracts are submitted? I could only figure out how to get it to do this for registered users, but  don't want to register a mailing list as a user...

**Comment by DirkHoffmann** [comment]
Users can subscribe to the iCal stream of a category or event (https://indico.cern.ch/category/XXX/events.ics?user_token=XXX e.g.). I don't say that it is a good replacement, but at least an existing workaround.

For the (technically unrelated) request about abstracts, better submit a separate ticket. (Purely personal advice, it's not me who triages them.)

**Comment by ThiefMaster** [comment]
I agree, mixing two unrelated things in the same issue is messy. Also, "is it possible"-type questions belong in the [forum](https://talk.getindico.io) IMHO
